File: Assignment35_1.c
// Write a program which accepts string from user and count number of capital characters.

#include<stdio.h>

int CountCapital(char *str)
{
    int iCount = 0;

    while(*str != '\0')
    {
        if((*str >= 'A') && (*str <= 'Z'))
        {
            iCount++;
            
        }
        str++;
    }
    return iCount;
}

int main()
{
    char Arr[] = {'\0'};
    int iRet = 0;

    printf("Enter a String: \n");
    scanf("%[^'\n']s",Arr);

    iRet = CountCapital(Arr);

    printf("Number of capital characters in string are: %d\n",iRet);

    return 0;
}

File: Assignment35_1.java
// Write a program which accepts string from user and count number of capital characters.

import java.util.Scanner;
class Logic
{
    public int CountCapital(String str)
    {
        char Arr[] = str.toCharArray();
        int iCount= 0;

        for(int i = 0; i < Arr.length; i++)
        {
            if((Arr[i] >= 'A') && (Arr[i] <= 'Z'))
            {
                iCount++;
            }
        }
        return iCount;
    }
}

class Assignment35_1
{
    public static void main(String[] args) 
    {
        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter a string: ");
        String sobj = scanobj.nextLine();

        Logic lobj = new Logic();
        int iRet = lobj.CountCapital(sobj);    
        
        System.out.println("Number of capital characters are: "+iRet);

    }
}